README.md

```diff
@@ -15,6 +15,7 @@ rabbitmq-c-users and cross-posted to rabbitmq-users.
  - <https://groups.google.com/forum/#!forum/rabbitmq-c-users>
  - <https://groups.google.com/forum/#!forum/rabbitmq-users>
 
+<a name=latest-stable-version></a>
```

Comment:
This isn't needed.

The change you made below should correctly link to the Latest Stable Version header.

Reply:
I didn't know about that. Thanks!